explorer/src/explorer/Params.hs

```diff
@@ -69,15 +67,7 @@ getNodeParams args@Args {..} systemStart = do
 
     npNetworkConfig <- intNetworkConfigOpts networkConfigOpts
 
-    let devBalanceDistr =
-            devBalancesDistr
-                (CLI.flatDistr commonArgs)
-                (CLI.richPoorDistr commonArgs)
-                (CLI.expDistr commonArgs)
-
-    let npGenesisCtx
-            | isDevelopment = devGenesisContext devBalanceDistr
-            | otherwise = genesisContextProduction
```

Comment:
A mistake to remove this? How do we control when a development genesis context is used?

Reply:
We don't need this anymore, I have a new change (https://issues.serokell.io/issue/CSE-217) that I'll create soon that uses `Pos.Client.CLI.getNodeParams`.

Reply:
we will generate a development genesis context like genesis for testnet from seed, hence we won't consider development mode separately and will remove all predefined dev fields